docs/PfSense.md

```diff
@@ -0,0 +1,20 @@
+In order to boot PfSense via PXE and use unattended install,
+you need the following:
+
+1. Extracted PfSense installation ISO exported via NFS.
+
+2. Relevant entry in DHCP server config:
+    `option root-path "/srv/tftp/images/pfsense";`
+   /srv/tftp/images/pfsense is the path to the extracted ISO.
+
+3. Relevant entries in PXELINUX config:
+```
+label PfSense
+ menu label PfSense
+ pxe images/pfsense/boot/pxeboot
+```
```

Comment:
@pkubaj why don't you add this entry to our configuation of PXE server ?

Reply:
sure